suite-native/intl/src/en.ts

```diff
@@ -491,6 +492,12 @@ export const en = {
                 failed: 'Device authenticity check failed: {error}',
             },
         },
+        wipeDevice: {
+            title: 'Wipe device',
+            subTitle: 'This will reset all of your device’s data. Proceed with caution.',
```

Comment:
tiny nit: 
```suggestion
            subtitle: 'This will reset all of your device’s data. Proceed with caution.',
```
To make it united with other occurrences.

Reply:
Good catch, fixed